docker-for-mac/edge-release-notes.md

```diff
@@ -11,6 +11,32 @@ This page contains information about Docker Desktop Edge releases. Edge releases
 For information about Stable releases, see the [Stable release notes](release-notes.md). For Docker Desktop system requirements, see
 [What to know before you install](install.md#what-to-know-before-you-install).
 
+## Docker Desktop Community 2.3.6.0
+2020-09-01
+
+> [Download](https://desktop.docker.com/mac/edge/TODO/Docker.dmg)
+
+### New
+
+- Docker Desktop introduces a new image scanning feature thanks to the partnership between Docker and Snyk.
+- Docker ECS plugin has been replaced by [ECS cloud integration](https://docs.docker.com/engine/context/ecs-integration/)
+- Docker UI:
+  - The images panel now has search and filters
+  - An image can now be pushed to Docker Hub from the drop-down menu
```

Comment:
You can now push an image to Docker Hub using the Remote repositories drop-down menu.